tests/payloads/networkapi/private_networks_delete_by_id.json

```diff
@@ -3,10 +3,12 @@
         "method": "DELETE",
         "path": "/networks/v1/private-networks/{id}",
         "pathParameters": {
-            "id": ["e6afba51-7de8-4080-83ab-0f915570659c"]
+            "id": [
+                "603f3b2cfcaf050643b89a4b"
+            ]
         }
     },
     "response": {
-        "statusCode": 204
+        "statusCode": 202
```

Comment:
Why did you change this to 202? Also why were these files reformatted?

Reply:
my mistake should have been 204. jsons were mainly changed because response body and result were either missing createdOn as part of body and where not in the right order so i had to reference the developers portal to fix them

Reply:
oh I mean the reformatting, like expanding the array of values for each query parameter - could these formatting changes be undone please?

Reply:
Some query string parameters are still being expanded, since we're rarely going to have multiple values for a single query parameter within tests, I think the one-liner approach is better.

Reply:
I applied your format wherever I could now. As discussed yesterday something like this should either be documented incase your sticking to some custom format, or else make use of a common reformater  so whoever maintains this after will know about this.

Reply:
Agreed.